src/animation.js

```diff
@@ -76,10 +82,21 @@ exports.requestAnimationFrame = function(callback, node) {
 /**
 * Cancels a requested animation callback with the specified id.
 *
+* @param {Number} id - The identifier of an animation request we wish to cancel.
+* @deprecated since 2.7.0
+* @public
+*/
+exports.cancelRequestAnimationFrame = function(id) {
```

Comment:
Seems reasonable enough - should we do the same for `requestAnimationFrame` as well?